java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1903,9 +1903,33 @@ public ColumnVector asTimestamp(DType timestampType, String format) {
    * @return A new vector allocated on the GPU.
    */
   public ColumnVector asStrings() {
+    if (type.isTimestamp()) {
+      return asStrings("%Y-%m-%d %H:%M:%S");
+    }
     return castTo(DType.STRING);
   }
 
+  /**
+   * Method to parse and convert a timestamp column vector to string column vector. A unix
+   * timestamp is a long value representing how many units since 1970-01-01 00:00:00:000 in either
+   * positive or negative direction. This mirrors the functionality spark sql's from_unixtime.
```

Comment:
I wouldn't call out the spark sql from_unixtime here.  The time format string is completely different from what is used here and it is likely to confuse some people.